examples/mutex/src/lib.rs

```diff
@@ -0,0 +1,42 @@
+#![no_std]
+
+use core::num::ParseIntError;
+use gstd::{exec, msg, prelude::*, ProgramId};
+use gstd_async::msg as msg_async;
+
+static mut PING_DEST: ProgramId = ProgramId([0u8; 32]);
+static MUTEX: gstd_async::mutex::Mutex<u32> = gstd_async::mutex::Mutex::new(0);
+
+const GAS_COST: u64 = 5_000_000;
```

Comment:
```suggestion
const GAS_LIMIT: u64 = 5_000_000;
```

It's known issue for examples: I think it's more correct to call it as GAS_LIMIT instead of GAS_COST